vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -160,6 +167,21 @@ def parse_pod(pod)
         end
       end
 
+      # NOTE: what we are trying to access here is the attribute:
+      #   pod.metadata.annotations.kubernetes.io/created-by
+      # but 'annotations' may be nil. The weird attribute name is
+      # generated by the JSON unmarshalling.
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
@simon3z Thanks for the comment.

Is this something we're in a position to create a test for?  If I'm new to this code and had to add "kubernetes.io/new-foo-thing", I'd git grep for "kubernetes.io" in manageiq to find other places we get related values and hopefully find a test that demonstrates how it's done.

We can do this as a followup if we don't have a way to test this... but if we're in here already, it would be nice to have a test for this since it's not idiomatic ruby to have a method name like this.